docs-website-src/content/blockers.md

```diff
@@ -149,3 +149,15 @@ These issues with the YUM repositories can cause ELevate to block your upgrade:
   * If YUM is in an unstable state (running `yum makecache` fails).
 
 If any unfinished yum transactions are detected, ELevate will attempt to complete them by executing `/usr/sbin/yum-complete-transaction --cleanup-only`. If this fails, ELevate will block you from beginning the upgrade process until you manually resolve any outstanding issues or transactions.
+
+#### Unsupported Repository
+
+By default, our stance on package repositories is that if they are not in a specific list of "vetted" repos,
+we will not allow upgrade while packages from said repository are installed.
+This is because of the following reasons:
+  * We have no way of knowing whether the repositories even have equivalent packages for the version the distribution we are upgrading to.
```

Comment:
Maybe:
"the version **of** the distribution"
instead of:
"the version the distribution"

Reply:
Filed DOCS-261 for phrase review internally